src/image/transform/__tests__/grey.js

```diff
@@ -93,9 +93,10 @@ describe('Grey transform', function () {
             ],
             {kind: 'GREYA'}
         );
+
         (function () {
-            image.grey().should.throw(/only be applied/);
-        });
+            image.grey();
+        }).should.throw(/only be applied/);
```

Comment:
@lpatiny `should.throw` must be called on the function, not inside! Otherwise nothing is executed.